### PR TITLE
Adds SPEC file for version 0.0.4 of the plugin

### PR DESCRIPTION
**Pull request by navidshaikh**
copr build https://copr.fedorainfracloud.org/coprs/nshaikh/vagrant-service-manager/build/168661/

### CODE REVIEW

**Comment by praveenkumar** [comment]
LGTM.

**Comment by LalatenduMohanty** [comment]
@navidshaikh can you please rebase the PR?

**Comment by navidshaikh** [comment]
@LalatenduMohanty : Rebased.

File: vagrant-service-manager.spec [hunk 1]
```diff
@@ -69,6 +69,7 @@ popd
 %{vagrant_plugin_libdir}
 %exclude %{vagrant_plugin_cache}
 %{vagrant_plugin_spec}
+%{vagrant_plugin_instdir}/plugins/guests/
```
Comment (LalatenduMohanty) [inline vagrant-service-manager.spec:72]: I am not sure if we have files in %{vagrant_plugin_dir}/\* related to plugins/guests . The question is do we really care about ruby code with respect to plugins/guests in the RPM. IMO if we have some files in %{vagrant_plugin_dir}/\* with respect to plugins/guests, we should remove them from the RPM.

Comment (praveenkumar) [inline vagrant-service-manager.spec:72]: As per https://fedoraproject.org/wiki/PackagingDrafts/Vagrant#Macros %{vagrant_plugin_instdir} should contain actual content of the plugin like Gemfile, Rakefile, tests ..etc. Plugin gem should be part of %{vagrant_plugin_dir} and code content should be part of %{vagrant_plugin_libdir}

**Comment by praveenkumar** [comment]
LGTM

**Pull request merged**